python/protos/test.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
```

Comment:
This looks like a nice test!

Just one thing missing in my opinion.

Protobuf uses relative imports to work.
Can you add one more file to the example that imports the test.proto  and then generate?
Because, if that works, then, we know for sure there is no import issue due to the folder structure inside the docker image